packages/web/src/common/store/queue/mobileSagas.ts

```diff
@@ -72,7 +73,10 @@ function* getTrackInfo(id: ID, uid: UID) {
     : track.cover_art
 
   const m3u8Gateways = gateways.concat(PUBLIC_IPFS_GATEWAY)
-  const m3u8 = generateM3U8Variants(track.track_segments, [], m3u8Gateways)
+  const m3u8 = generateM3U8Variants({
```

Comment:
don't we want to do this inside the component on the native side? or is that not necessary now?

Reply:
yeah we do, but this is just for backwards compat on main!

Reply:
ohhh i didn't recognize the diff base here. sounds great